docs/book/user-guide/advanced-guide/environment-management/environment-variables.md

```diff
@@ -0,0 +1,89 @@
+---
+description: How to control ZenML behavior with environmental variables.
+---
+
+# ZenML Environment Variables
+
+There are a few pre-defined environmental variables that can be used to control 
+the behavior of ZenML. See the list below with default values and options:
+
+## Logging verbosity
```

Comment:
Recently we had a few issues regarding the step logs storage, that's why it might make sense to include the `ENV_ZENML_DISABLE_STEP_LOGS_STORAGE` in the list as well.

Reply:
Done